src/python/pants/base/hash_utils.py

```diff
@@ -154,13 +117,10 @@ def encode(self, o):
         return super().encode(self.default(o))
 
 
-def json_hash(
-    obj: Any, digest: Optional[Digest] = None, encoder: Optional[Type[json.JSONEncoder]] = None
-) -> str:
+def json_hash(obj: Any, encoder: Optional[Type[json.JSONEncoder]] = None) -> str:
```

Comment:
I don't understand why we would remove the `digest` option here when it's a very small amount of code and defaults to `None`?

Reply:
What is the reason to keep any unused code at all ever? There is no imminent planned use.